rust_src/src/windows.rs

```diff
@@ -1967,4 +1967,30 @@ pub fn set_window_fringes_lisp(
     }
 }
 
+/// Return pixel width of window WINDOW before last size changes.
+/// WINDOW must be a valid window and defaults to the selected one.
+///
+/// The return value is the pixel width of WINDOW at the last time
+/// `window-size-change-functions' was run.  It's zero if WINDOW was made
+/// after that.
+#[lisp_fn(min = "0")]
+pub fn window_pixel_width_before_size_change(window: LispWindowValidOrSelected) -> EmacsInt {
+    LispWindowRef::from(window)
```

Comment:
I think we should use `let window: LispWindowRef = window.into();` to make things consistent.

Reply:
Makes sense.